projects/plugins/jetpack/extensions/blocks/payments-intro/index.js

```diff
@@ -9,10 +9,10 @@ import { currencyDollar } from '@wordpress/icons';
  * Internal dependencies
```

Comment:
This is unrelated to your actual changes. These would be WordPress dependencies instead of External dependencies. If it isn't too much effort maybe we could change this.